RunningOrder/InlineEditableList.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  InlineTexField.swift
+//  RunningOrder
+//
+//  Created by Lucas Barbero on 20/07/2020.
+//  Copyright © 2020 Worldline. All rights reserved.
+//
+
+import SwiftUI
+import AppKit
+
+/// Display an inline editable list of string element
+struct InlineEditableList: View { // Change name ???
+
+    let title: String
+
+    let placeholder: String
+
+    @Binding var values: [String]
+    @State private var hovered = false
+
+    var body: some View {
+        VStack {
+
+            HStack {
+                Text(title).foregroundColor(.gray).padding(.all, 6)
+                Spacer()
+                if hovered {
+                    AddButton(action: addTextfieldValue)
+                        .padding(.horizontal, 10)
+                }
+            }
+
+            ForEach(values.indices, id: \.self) { index in
+                ZStack(alignment: .trailing) {
+                    FocusableTextField(placeholder, value: Binding(
```

Comment:
no binding for $values[index] ? :P

Reply:
This method cause some 'Fatal error: Index out of range'